dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/utils/ContextUtils.java

```diff
@@ -344,6 +344,21 @@ public static boolean isAcceptCsvGzip( HttpServletRequest request )
                 && request.getHeader( "Accept" ).contains( "application/csv+gzip" )));
     }
 
+    /**
+     * Indicates whether the given requests indicates that it accepts a
+     * compressed response.
+     *
+     * @param request the HttpServletRequest.
+     * @return whether the given requests indicates that it accepts a compressed
+     *         response.
+     */
+    public static boolean isAcceptCsvZip( HttpServletRequest request )
+    {
+        return request != null && ((request.getPathInfo() != null && request.getPathInfo().endsWith( ".zip" ))
+            || (request.getHeader( "Accept" ) != null
```

Comment:
Use Spring HttpHeaders constants for header names instead of strings ("Accept")?